README.rst

```diff
@@ -52,24 +55,25 @@ The default Make target generates HTML for all PEPs.
 If you don't have Make, use the ``pep2html.py`` script directly.
 
 
-Generating HTML for python.org
+Generating HTML for Python.org
 ==============================
 
-python.org includes its own helper modules to render PEPs as HTML, with
+Python.org includes its own helper modules to render PEPs as HTML, with
 suitable links back to the source pages in the version control repository.
 
-These can be found at https://github.com/python/pythondotorg/tree/main/peps
+These can be found `in the Python.org repository
```

Comment:
I'd probably keep `python.org` lowercase (caps for titles and sentence starts are okay).

Reply:
Okay, reverted all non-title and non-sentance-starting instances.